rpcs3/Emu/Audio/ALSA/ALSABackend.h

```diff
@@ -0,0 +1,30 @@
+#pragma once
+
+#ifdef HAVE_ALSA
+
+#include "Emu/Audio/AudioBackend.h"
+
+#include <alsa/asoundlib.h>
+
+class ALSABackend : public AudioBackend
+{
+	snd_pcm_t* s_tls_handle{nullptr};
```

Comment:
As these are no longer static and thread local, I think the prefixes should be altered to reflect that.